### PR TITLE
add opacity transformatin

### PR DESCRIPTION
**Pull request by violetviolinist**

### CODE REVIEW

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 1]
```diff
@@ -830,6 +830,32 @@ URL - [https://ik.imagekit.io/demo/img/logo-white\_SJwqB4Nfe.png?tr=bg-F00000](h
 
 ![](https://ik.imagekit.io/demo/img/logo-white\_SJwqB4Nfe.png?tr=bg-F00000)
 
+### Opacity - (o)
+
+It is used to specify the opacity level of the output image. A non-tarnsparent image can be made semi-transparent by specifying an opacity level that is less than 100. If an image already has a transparency, then it is further multiplied based on the specified value. It accepts numerical values ranging from 0 to 100, where 0 would produce a completely transparent image, and 100 would have no effect. If a background (`bg`) color is specified along with opacity, then that color is blended with the resultant image, leading to a non-transparent (opaque) image, but with the `bg` color mixed in. Alpha component, if present in the `bg` value, will be ignored.
```
Comment (imagekitio) [inline features/image-transformations/resize-crop-and-other-transformations.md]: Typo in `non-tarnsparent`. See if whole thing can be reworded or split into two para.

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 2]
```diff
@@ -830,6 +830,32 @@ URL - [https://ik.imagekit.io/demo/img/logo-white\_SJwqB4Nfe.png?tr=bg-F00000](h
 
 ![](https://ik.imagekit.io/demo/img/logo-white\_SJwqB4Nfe.png?tr=bg-F00000)
```
Comment (imagekitio) [inline features/image-transformations/resize-crop-and-other-transformations.md:831]: Need to update tables of supported parameter in overlay or other pages 
https://docs.imagekit.io/features/image-transformations/overlay-using-layers

File: features/image-transformations/resize-crop-and-other-transformations.md (hunk 1 above)
Comment (violetviolinist) [reply features/image-transformations/resize-crop-and-other-transformations.md]: done

File: features/image-transformations/resize-crop-and-other-transformations.md (hunk 2 above)
Comment (violetviolinist) [reply features/image-transformations/resize-crop-and-other-transformations.md:831]: added

**Pull request merged**